.github/workflows/CI.yml

```diff
@@ -1,7 +1,11 @@
 name: CI
 on:
-  - push
-  - pull_request
+  push:
+    branches: "main"
+    tags: ["*"]
+  pull_request:
+  release:
```

Comment:
Simply asking if this is needed? I mean, we have in this repo the "master" branch (old scheme) rather than the "main" branch (new scheme).